quartz/cfg.ts

```diff
@@ -41,6 +41,7 @@ export type Analytics =
 
 export interface GlobalConfiguration {
   pageTitle: string
+  pageTitleSuffix: string
```

Comment:
this would break current configuration if users don't configure `pageTitleSuffix`. It should be
```suggestion
  pageTitleSuffix?: string
```

Reply:
Oh, okay. I already had it preconfigurd with a value of `""` in `quartz.config.ts`. You're right though, it should be an optional config entry.